docs/guides/LiveCode Builder Language Reference.md

```diff
@@ -575,6 +575,64 @@ includes the specification of function signature, according to the
 [standard rules for forming these](http://journals.ecs.soton.ac.uk/java/tutorial/native1.1/implementing/method.html) 
 when calling the JNI.
 
+The function `interface` may be used on Android to create an interface 
```

Comment:
We probably need to say something about threading. As it stands, you have to be careful to not to try and use callbacks on any thread other than the UI thread or engine thread or bad things will (in all likelyhood) happen.